### DISCUSSION TITLE
Adding support for new browsers

### DISCUSSION THREAD

**Discussion by jgkaplan** [will-stone/browserosaurus#535] [Ideas]
Is it possible to move the browser list (from apps.ts) to some sort of external format (maybe a js or json file stored in Browsersaurus.app/Contents/Resources, so it can be extended without rebuilding from source? I was looking into this, but I'm not quite sure how electron-forge bundles resources. A feature to drag&drop a browser onto the settings window to automatically add it could cool.

**Comment by will-stone** [comment]
If we don’t want icons then yeah, we could totally move it to `~/config/browserosaurus.json`. But that’s the rub, we all love those little icons 😎 If you can find a way to get the icon to display just by using the bundle ID, then we may able to move the config.

**Reply by jgkaplan** [reply]
Ah I see. Thanks for the reply! I'll look in to it.

**Reply by will-stone** [reply]
I think I've figured out how to get the icons from the system! 🥳 So I'll probably do a release with this first, and then I'll figure out the best way to expose the config 🙂
